rating/src/main/java/com/yas/rating/service/RatingService.java

```diff
@@ -59,9 +69,17 @@ public RatingVm createRating(RatingPostVm ratingPostVm) {
         return RatingVm.fromModel(savedRating);
     }
 
-    public Double calculateAverageStar(Long productId){
+    public ResponeStatusVm deleteRating(long ratingId) {
+        Rating rating = ratingRepository.findById(ratingId)
+                .orElseThrow(() -> new NotFoundException(Constants.ERROR_CODE.RATING_NOT_FOUND, ratingId));
+
+        ratingRepository.delete(rating);
+        return new ResponeStatusVm("Delete Rating", " Success", HttpStatus.OK.toString());
```

Comment:
Can you extract "Success" to constant and then reuse it ?

Reply:
Thank you, I will update